learn/tanssi/services-payment.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Services Payment
```

Comment:
```suggestion
title: Block Production Services Payment
```

For now, let's add Block Production to the beginning of the title so it's a bit more descriptive in search engine results